decidim-assemblies/spec/presenters/decidim/assemblies/admin_log/assemblies_type_presenter_spec.rb

```diff
@@ -5,7 +5,7 @@
 module Decidim
   module Assemblies
     module AdminLog
-      describe AssembliesTypePresenter, type: :helper do
+      describe AdminLog::AssembliesTypePresenter, type: :helper do
```

Comment:
Maybe it's better to simply specify the whole class & module names?

```
describe Decidim::Assemblies::AdminLog::AssembliesTypePresenter do
```

Same for the rest of the files

Reply:
@mrcasals At some point I got Rubocop violation for doing that but I'm not sure if the rules have changed.

Reply:
@mrcasals I tried to change it like this and this is what I get:

![image](https://user-images.githubusercontent.com/864340/98987528-f7bcc900-252e-11eb-9f71-7ef830a3408c.png)

Reply:
That's weird, because we have specs similar to what you post:

https://github.com/decidim/decidim/blob/d113f0690fb589961d3fd76f7bfe8ea35b2e0579/decidim-assemblies/spec/presenters/decidim/assemblies/admin_log/value_types/member_position_presenter_spec.rb#L5

Maybe your Rubocop is not getting the Decidim config for some reason?

Reply:
No, it's using the correct config. Other cops work totally fine.

It seems to be an issue with the VSCode extension:
https://github.com/rubyide/vscode-ruby/issues/625

Althrough I don't know where it's coming from because what that extension is doing is running the same rubocop that I can run on the command line...

Reply:
@mrcasals It's now changed as you proposed.

I assume the issue is in rubocop-rspec, I reported it here with further details:
https://github.com/rubocop-hq/rubocop-rspec/issues/1091

The result of that cop depends where rubocop is run from which doesn't seem right to me. The CI runs it in the root of the project when it works correctly. The VSCode extension runs it in the same folder where the file is located at when it does not work.

Reply:
Ah, nice catch, thanks!!